src/main/java/io/jenkins/plugins/analysis/core/steps/IssuesRecorder.java

```diff
@@ -1131,6 +1161,10 @@ public FormValidation doCheckReferenceJobName(@QueryParameter final String refer
             return model.validateJob(referenceJobName);
         }
 
+        public FormValidation doCheckReferenceBuildId(@QueryParameter final String referenceBuildId) {
```

Comment:
remove this method as this field cannot be part of the validation (you are returning true in any case)